### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by earakelianEPAM**

### CODE REVIEW

**Review by obitkin — CHANGES_REQUESTED** [review]
- Rename repo to NameSurname
- Unite 2 maven project into 1
- Remove useless .idea files from git
- Remove target dir from git
- Add checkstyle plugin
- Fix checkstyle errors
Also, tests are very simple. You haven't found any problem. It's not requirements but it's possible to make it better

**Comment by earakelianEPAM** [comment]
Tests are running successfully with maven command now

**Review by obitkin — CHANGES_REQUESTED** [review]
- Sorry, but the project structure is wrong. Check how it should look like here: https://github.com/jenkins-docs/simple-java-maven-app
- ~~You should add testng suite for all tests~~

**Comment by obitkin** [comment]
Also, don't use github web for commit push. It's a very bad way to do the changes. For example instead of 1 commit for file deleting I see hundreds of them

**Review by obitkin — CHANGES_REQUESTED** [review]
- Remove FirtsMavenProject directory, move files from it to repo root
- Remove .idea files
- Remove target files(it's not a source code)
- Fix check style errors

File: src/test/java/com/epam/tc/hw1/DivideTest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        };
+    }
+
+    @Test(dataProvider = "divCorrectData")
+    public void divDDtTest(double a, double b, double res) {
+        var act = calculator.div(a, b);
+        assertThat(res).as("Wrong calculation").isEqualTo(act);
+
+    }
+}
```
Comment (obitkin) [inline src/test/java/com/epam/tc/hw1/DivideTest.java]: Add an empty line at the end of this file to fix check style error

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    <groupId>com.epam.tc</groupId>
+    <artifactId>Edgar_Arakelian</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
```
Comment (obitkin) [inline pom.xml]: When I try to run tests via `mvn clean test` command I don't see any code or test compilation, or tests run. 

Try to find and fix the reason of this (it's in this part of pom file)

**Review by obitkin — CHANGES_REQUESTED** [review]

**Review by obitkin — APPROVED** [review]
Approved

**Pull request merged**